gate_fixtures/invoice_store.py

```
@@ -0,0 +1,39 @@
+
+def connect():
+    return sqlite3.connect(os.environ.get("INVOICE_DB", "invoices.db"))
+
+
+def find_invoice(customer_id, status):
+    conn = connect()
+    cur = conn.cursor()
+    query = (
+        "SELECT id, amount, currency FROM invoices "
+        "WHERE customer_id = '" + str(customer_id) + "' "
+        "AND status = '" + str(status) + "'"
```

Ruff (0.16.1)

[error] Possible SQL injection vector through string-based query construction

(S608)